java/source/org/libreoffice/ide/eclipse/java/registration/RegistrationHandler.java.tpl

```diff
@@ -96,7 +96,8 @@ public class RegistrationHandler '{'
             }
             if (services == null) '{'
                 services = new String[] '{'implementation};
-                System.err.println(String.format("No <%s> static field, defaulting to: '{'%s}", fieldName, implementation));
+                String template = "No <%s> static field, defaulting to: '{'%s}";
```

Comment:
This seems unrelated? Should be a separate commit.

Reply:
I'm preparing another one...